src/modules/api.js

```diff
@@ -0,0 +1,61 @@
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/nslYEPpt500fFqeTXs8e';
+
+async function submitScore(name, score) {
+  const response = await fetch(`${API_URL}/scores`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+  });
+  const result = await response.json();
+  return result;
+}
```

Comment:
- [ ] According to the requirements we are supposed to use ES6 this includes usage of arrow functions instead of the `function keyword`, please refactor this function to an arrow function.